### PR TITLE
msx2_flop.xml: Add alt_title, serial, gtin, and part_id meta information and notes.

### PR DESCRIPTION
**Pull request by wilbertpol**

### CODE REVIEW

**Comment by wilbertpol** [comment]
I have encountered several other things to improve in this software list but to make the PR handling simpler I have only added (and updated a few) metadata and notes.
I did make one non-metadata change and that was moving the 2 entries for mikkoku from the end of the list to the middle.

File: hash/msx2_flop.xml [hunk 1]
```diff
@@ -9565,6 +11606,8 @@ The following floppies came with the machines.
 		<description>Record of Lodoss War - Haiiro no Majo (Japan)</description>
 		<year>1989</year>
 		<publisher>Humming Bird Soft</publisher>
+		<info name="alt_title" value="ロードス島戦記"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:11609]: Should the subtitle be in the `alt_title` as well here (灰色の魔女)?  It is for the sequel.

File: hash/msx2_flop.xml [hunk 2]
```diff
@@ -9917,6 +12026,8 @@ The following floppies came with the machines.
 		<description>Rune Master III (Japan, alt)</description>
 		<year>1991</year>
 		<publisher>Compile</publisher>
+		<info name="alt_title" value="ルーンマスター三国英傑伝"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:12029]: Why is the subtitle in the description of the parent but not the clones for this game?

File: hash/msx2_flop.xml [hunk 3]
```diff
@@ -10240,12 +12412,16 @@ The following floppies came with the machines.
 		<description>Schwarzschild II (Japan)</description>
 		<year>1990</year>
 		<publisher>Kogado</publisher>
+		<info name="alt_title" value="シュヴァルツシルト II 帝國ノ背信"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:12415]: The subtitle is in the `alt_title` but not the description for this game now.

File: hash/msx2_flop.xml [hunk 4]
```diff
@@ -10294,6 +12480,7 @@ The following floppies came with the machines.
 		<description>Seikimatsu Fly Battler (Japan)</description>
 		<year>1993</year>
 		<publisher>MSX-FAN</publisher>
+		<info name="alt_title" value="世紀末FLY BATTLER弥七"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:12483]: The title in the description appears to be incomplete (missing 弥七 at the end – I assume it’s the protagonist’s name Yashichi).

File: hash/msx2_flop.xml [hunk 5]
```diff
@@ -11203,38 +13564,51 @@ The following floppies came with the machines.
 		<description>The Tower? of Cabin - Cabin Panic (Japan)</description>
 		<year>1992</year>
 		<publisher>Micro Cabin</publisher>
+		<info name="alt_title" value="THE TOWER OF CABIN?～キャビンパニック～"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:13567]: Is the question mark supposed to be in a different place in the description and `alt_title`?

File: hash/msx2_flop.xml [hunk 6]
```diff
@@ -11299,17 +13683,23 @@ The following floppies came with the machines.
 		<description>Tir-nan-óg - Dana no Matsuei (Japan)</description>
 		<year>1990</year>
 		<publisher>System Soft</publisher>
+		<info name="alt_title" value="ティルナノーグ"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:13686]: Is the title correct?  I don’t think Tir-nan-óg: Descendents of Danaan was released for MSX2.  Forbidden Tower released for MSX2: https://www.generation-msx.nl/software/system-soft/tir-nan-og/release/1388/

The title appears to be “禁断の塔 ティル・ナ・ノーグ” (Kindan no Tou: Tir-na-nóg).

File: hash/msx2_flop.xml [hunk 7]
```diff
@@ -12037,12 +14588,14 @@ The following floppies came with the machines.
 		<description>Wizardry - Proving Grounds of Mad Overload (Japan)</description>
 		<year>1987</year>
 		<publisher>ASCII</publisher>
+		<info name="alt_title" value="ウィザードリィ"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:14591]: The Japanese title is “ウィザードリィ 狂王の試練場”.

File: hash/msx2_flop.xml [hunk 8]
```diff
@@ -12580,12 +15230,15 @@ The following floppies came with the machines.
 		<description>XZR - Hakai no Guuzou (Japan)</description>
 		<year>1988</year>
 		<publisher>Telenet Japan</publisher>
+		<info name="alt_title" value="エグザイル"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:15233]: The full Japanese title is “エグザイル 破戒の偶像”

File: hash/msx2_flop.xml [hunk 9]
```diff
@@ -12612,60 +15268,78 @@ The following floppies came with the machines.
...
 			</dataarea>
 		</part>
 	</software>
 
+	<!-- Release contained 2 floppies and a CD. -->
 	<software name="xzr2">
 		<description>XZR II (Japan)</description>
 		<year>1988</year>
 		<publisher>Telenet Japan</publisher>
+		<info name="alt_title" value="エグザイルII"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:15291]: The title of this game is “エグザイルII 完結編”, (XZR II: Kanketsu-hen).

File: hash/msx2_flop.xml [hunk 10]
```diff
@@ -12827,7 +15529,10 @@ The following floppies came with the machines.
 	<software name="ysa" cloneof="ys">
 		<description>Ys - Ancient Ys Vanished Omen (Japan, alt)</description>
 		<year>1987</year>
-		<publisher>Falcom</publisher>
+		<publisher>Sony</publisher>
+		<info name="alt_title" value="イース"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:15533]: Is the subtitle “Ancient Ys Vanished Omen” actually used anywhere?  Different releases used “Ancient Ys Vanished” and “The Vanished Omens”.  As far as I know, the MSX2 box art says “Ancient Ys Vanished” (the original PC-8801 box art was used).

File: hash/msx2_flop.xml [hunk 11]
```diff
@@ -1016,7 +1185,11 @@ The following floppies came with the machines.
 		<description>Arcusyu - Kagerou no Jidai wo Koete (Japan)</description>
 		<year>1989</year>
 		<publisher>WolfTeam</publisher>
+		<info name="alt_title" value="あーくしゅ"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:1188]: Looking at the images again of the box and disk (https://www.generation-msx.nl/software/wolfteam/arcusyu---arcus-arch/release/1298/) it looks like our original title was incorrect and we should be using something like "あーくしゅ[アークス番外編]" and "Arcusyu - a-kusu bangaihen" ?

File: hash/msx2_flop.xml [hunk 12]
```diff
@@ -1281,6 +1503,7 @@ The following floppies came with the machines.
 		<description>Bishoujo Shashinkan - Moving School + Studio Cut (Japan)</description>
 		<year>1987</year>
 		<publisher>Hard</publisher>
+		<info name="alt_title" value="美少女写真館ダブルヴィジョン"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:1506]: Thanks, that is indeed much better and also matches the full title on the box.

File: hash/msx2_flop.xml (hunk 11 above)
[the remark this answers is not in the document]
Comment (angelosa) [reply hash/msx2_flop.xml:1188]: https://refuge.tokyo/pc9801/pc98/00179.html fwiw (which implies that needs to be fixed in pc98.xml and x68k_flop.xml too, both with "Arcshu - Kagerou no Jidai o Koete")

File: hash/msx2_flop.xml [hunk 13]
```diff
@@ -6745,28 +8153,35 @@ The following floppies came with the machines.
...
 			<dataarea name="flop" size="737280">
 				<rom name="kudokikata oshiemasu (1988)(hard)(jp)[a].dsk" size="737280" crc="954dfed0" sha1="da72037faa0c20746864c31a8a323d186dd4bec4"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<!-- Contains user data? One of the saves games is called STOPGAME. -->
 	<software name="kugyokud">
-		<description>Kugyokuden (Japan)</description>
+		<description>Ku Gyoku Den (Japan)</description>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:8167]: It was printed on the box with the spaces: https://www.generation-msx.nl/software/tecno-soft/legendly-nine-gems/release/930/

File: hash/msx2_flop.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:11609]: Probably better, yeah. The 灰色の魔女 is on the title screen and on the box, but not on the disk labels or the manual.

File: hash/msx2_flop.xml (hunk 7 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:14591]: The "狂王の試練場" part is not on the front of the box or on the disk labels. Looking at a bad image of the box that part does seem to be on the side of the box. Thanks.

File: hash/msx2_flop.xml (hunk 10 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:15533]: Yes, on the title screen:

File: hash/msx2_flop.xml [hunk 14]
```diff
@@ -1465,46 +1726,57 @@ The following floppies came with the machines.
 		<description>Burai II (Japan, alt)</description>
 		<year>1992</year>
 		<publisher>Brother Industries</publisher>
+		<info name="alt_title" value="ＢＵＲＡＩ下巻完結編"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:1729]: Yes, it should

File: hash/msx2_flop.xml [hunk 15]
```diff
@@ -1377,32 +1618,41 @@ The following floppies came with the machines.
 		<description>Burai Joukan (Japan, alt)</description>
 		<year>1989</year>
 		<publisher>Riverhill Soft</publisher>
+		<info name="alt_title" value="ＢＵＲＡＩ上巻"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:1621]: Changed all the 'Burai's to 'BURAI'

File: hash/msx2_flop.xml [hunk 16]
```diff
@@ -1861,6 +2215,8 @@ The following floppies came with the machines.
 		<description>Crescent Moon (Japan)</description>
 		<year>1990</year>
 		<publisher>Alicesoft</publisher>
+		<info name="alt_title" value="クレセントムーンがぁる"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:2218]: Thanks, added 'Girl'

File: hash/msx2_flop.xml [hunk 17]
```diff
@@ -2844,17 +3365,24 @@ The following floppies came with the machines.
 		<description>Disc Station - Soukan Junbi-0-Go (Japan)</description>
 		<year>1988</year>
 		<publisher>Compile</publisher>
+		<info name="alt_title" value="ディスクステーション創刊準備号"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:3368]: Must have been a typo from hitting both the - and 0 keys, fixed.

File: hash/msx2_flop.xml [hunk 18]
```diff
@@ -2866,12 +3394,17 @@ The following floppies came with the machines.
 		<description>Disc Station - Dai-2-Go (Japan)</description>
 		<year>1988</year>
 		<publisher>Compile</publisher>
+		<info name="alt_title" value="ディスクステーション２号"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:3397]: Thanks, fixed

File: hash/msx2_flop.xml [hunk 19]
```diff
@@ -7148,6 +8613,8 @@ The following floppies came with the machines.
 		<description>Love Chaser (Japan, alt)</description>
 		<year>1987</year>
 		<publisher>Champion Soft</publisher>
+		<info name="alt_title" value="美少女アドベンチャー"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:8616]: Thanks, updated description and alt_title.

File: hash/msx2_flop.xml [hunk 20]
```diff
@@ -7294,27 +8774,57 @@ The following floppies came with the machines.
...
+		<description>Maison Ikkoku (Japan, alt)</description>
+		<year>1987</year>
+		<publisher>Micro Cabin</publisher>
+		<info name="alt_title" value="めぞん一刻"/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="maison ikkoku (1987)(micro cabin)(jp)[a].dsk" size="737280" crc="a5f3333d" sha1="9223aba78d22a2ddba671c4b0be4649e2b5148fb"/>
+			</dataarea>
+		</part>
+	</software>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:8820]: Thanks, fixed

File: hash/msx2_flop.xml [hunk 21]
```diff
@@ -9317,22 +11297,28 @@ The following floppies came with the machines.
 		<description>Rance (Japan)</description>
 		<year>1989</year>
 		<publisher>Alicesoft</publisher>
+		<info name="alt_title" value="RANCE 光をもとめて"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:11300]: Thanks, added the subtitle to the description

File: hash/msx2_flop.xml [hunk 22]
```diff
@@ -9369,27 +11361,35 @@ The following floppies came with the machines.
 		<description>Rance II (Japan)</description>
 		<year>1990</year>
 		<publisher>Alicesoft</publisher>
+		<info name="alt_title" value="RANCE II - 反逆の少女達"/>
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx2_flop.xml:11364]: Thanks, added the subtitle to the description

File: hash/msx2_flop.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:12029]: I hadn't noticed that, thanks. Fixed.

File: hash/msx2_flop.xml (hunk 3 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:12415]: Also added the subtitle to the description now.

File: hash/msx2_flop.xml (hunk 4 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:12483]: Added the name to the description.

File: hash/msx2_flop.xml (hunk 8 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:15233]: Thanks, fixed.

File: hash/msx2_flop.xml (hunk 9 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:15291]: Thanks, fixed.

File: hash/msx2_flop.xml (hunk 5 above)
Comment (angelosa) [reply hash/msx2_flop.xml]: the_tower_of_cabin

File: hash/msx2_flop.xml (hunk 11 above)
[the remark this answers is not in the document]
Comment (cuavas) [reply hash/msx2_flop.xml]: > Looking at the images again of the box and disk (https://www.generation-msx.nl/software/wolfteam/arcusyu---arcus-arch/release/1298/) it looks like our original title was incorrect and we should be using something like "あーくしゅ[アークス番外編]" and "Arcusyu - a-kusu bangaihen" ?

You don’t transliterate sound extension as hyphens.  The “あーく” is a transliteration of “arc” – it’s.  If you assume the title on the box there is correct, the description should be “Arcshu: Arcs Bangai-hen” and the Japanese title should be “あーくしゅ［アークス番外編］”.

**Comment by cuavas** [comment]
There are probably more metadata errors lurking.  I’ll merge this and fix a couple of titles myself.  The rest can wait.

**Pull request merged**